src/modules/dashboard/components/CreatePaymentDialog/CreatePaymentDialogForm.tsx

```diff
@@ -0,0 +1,394 @@
+                          size: 'small',
+                          theme: 'grey',
+                        }}
+                        value={fromDomainTokenBalance || 0}
+                        unit={getTokenDecimalsWithFallback(
+                          selectedToken && selectedToken.decimals,
+                        )}
+                        truncate={3}
+                      />
+                    ),
+                    symbol: (selectedToken && selectedToken.symbol) || '???',
+                  }}
+                />
+              </div>
+            )}
+          </div>
+        </div>
+      </DialogSection>
+      <DialogSection>
+        <div className={styles.singleUserContainer}>
+          <SingleUserPicker
+            appearance={{ width: 'wide' }}
+            data={subscribedUsers}
+            label={MSG.to}
+            name="toAssignee"
+            filter={filterUserSelection}
+            renderAvatar={supRenderAvatar}
+            disabled={!userHasPermission}
+          />
+        </div>
+      </DialogSection>
+      <DialogSection>
+        <div className={styles.tokenAmount}>
+          <Input
+            label={MSG.amount}
+            name="amount"
+            appearance={{
+              theme: 'minimal',
+              align: 'right',
+            }}
+            formattingOptions={{
+              delimiter: ',',
+              numeral: true,
+              numeralDecimalScale: getTokenDecimalsWithFallback(
+                selectedToken && selectedToken.decimals,
+              ),
+            }}
+            disabled={!userHasPermission}
+          />
+          <div className={styles.tokenAmountSelect}>
+            <Select
+              label={MSG.token}
+              options={tokenOptions}
+              name="tokenAddress"
+              elementOnly
+              appearance={{ alignOptions: 'right', theme: 'grey' }}
+              disabled={!userHasPermission}
+            />
+          </div>
+          {values.tokenAddress === AddressZero && (
+            <div className={styles.tokenAmountUsd}>
+              <EthUsd
+                appearance={{ theme: 'grey', size: 'small' }}
+                value={
+                  /*
+                   * @NOTE Set value to 0 if amount is only the decimal point
+                   * Just entering the decimal point will pass it through to EthUsd
+                   * and that will try to fetch the balance for, which, obviously, will fail
+                   */
+                  values.amount && values.amount.length && values.amount !== '.'
+                    ? values.amount
+                    : 0
+                }
+              />
+            </div>
+          )}
+        </div>
+      </DialogSection>
+      <DialogSection>
+        <div className={styles.textAreaSection}>
+          <Textarea
+            appearance={{ resizable: 'vertical', colorSchema: 'grey' }}
+            label={MSG.reason}
+            name="reason"
```

Comment:
The name is `annotation` for this in the contracts.

It's wise to keep the same naming convention so that we don't get things mixed up. 

Please change it to:
```suggestion
            name="annotation"
```